StreetSmartArcMap/Forms/StreetSmartConfigurationForm.cs

```diff
@@ -142,7 +144,15 @@ private void LoadAPISettings()
 
         private void LoadGeneralSettings()
         {
-            nudOverlayDrawDistance.Value = Config.OverlayDrawDistanceInMeters;
+            var selectedSRS = (SpatialReference)cbCycloramaSRS.SelectedItem;
+            switch (selectedSRS.Units)
+            {
+                case "ft":
+                    nudOverlayDrawDistance.Maximum = (decimal)Math.Round((double)nudOverlayDrawDistance.Maximum * 3.280839895, 0);
+                    nudOverlayDrawDistance.Value = (decimal)Math.Round(Config.OverlayDrawDistanceInMeters * 3.280839895, 0);
+                    break;
+                default: break;
```

Comment:
if the SelectedSRS == null then the switch (selectedSRS.Units) crashes.
![image](https://user-images.githubusercontent.com/5754358/74339082-a4e99180-4da3-11ea-84ec-b95a4841c7c8.png)